prod/westeurope/linux/appbackendl1/app_service_slot_staging/terragrunt.hcl

```diff
@@ -0,0 +1,245 @@
+
+    // REDIS
+    REDIS_URL      = dependency.redis.outputs.hostname
+    REDIS_PORT     = dependency.redis.outputs.ssl_port
+    REDIS_PASSWORD = dependency.redis.outputs.primary_access_key
+
+    // PUSH NOTIFICATIONS
+    ALLOW_NOTIFY_IP_SOURCE_RANGE = dependency.subnet_fn3services.outputs.address_prefix
+
+    // LOCK / UNLOCK SESSION ENDPOINTS
+    ALLOW_SESSION_HANDLER_IP_SOURCE_RANGE = dependency.subnet_funcadmin_r3.outputs.address_prefix
+
+    // PAGOPA
+    PAGOPA_API_URL_PROD = "https://${dependency.app_service_pagopaproxyprod.outputs.default_site_hostname}"
+    PAGOPA_API_URL_TEST = "https://${dependency.app_service_pagopaproxytest.outputs.default_site_hostname}"
+    PAGOPA_BASE_PATH    = "/pagopa/api/v1"
+
+    // MYPORTAL
+    MYPORTAL_BASE_PATH = "/myportal/api/v1"
+
+    // BPD
+    BPD_BASE_PATH = "/bpd/api/v1"
+
+    SPID_LOG_QUEUE_NAME                = dependency.storage_queue_spid_logs.outputs.name
+    SPID_LOG_STORAGE_CONNECTION_STRING = dependency.storage_account_logs.outputs.primary_connection_string
+
+    NOTIFICATIONS_QUEUE_NAME                = dependency.notification_queue.outputs.name
+    NOTIFICATIONS_STORAGE_CONNECTION_STRING = dependency.notification_storage_account.outputs.primary_connection_string
+
+    // USERSLOGIN
+    USERS_LOGIN_STORAGE_CONNECTION_STRING = dependency.storage_account_logs.outputs.primary_connection_string
+    USERS_LOGIN_QUEUE_NAME                = dependency.storage_queue_users_login.outputs.name
+
+    // Feature flags
+    FF_BONUS_ENABLED = 1
+
+    TEST_LOGIN_FISCAL_CODES = "AAAAAA00A00A000B"
+
+    # No downtime on slots swap
+    WEBSITE_ADD_SITENAME_BINDINGS_IN_APPHOST_CONFIG = 1
+
+    JWT_SUPPORT_TOKEN_ISSUER     = "app-backend.io.italia.it"
+    JWT_SUPPORT_TOKEN_EXPIRATION = 604800
+  }
+
+  app_settings_secrets = {
+    key_vault_id = dependency.key_vault.outputs.id
+    map = {
+      // SPID
+      SAML_CERT = "appbackend-SAML-CERT"
+      SAML_KEY  = "appbackend-SAML-KEY"
+
+      // FUNCTIONS
+      API_KEY       = "funcapp-KEY-APPBACKEND"
+      BONUS_API_KEY = "funcbonus-KEY-APPBACKEND"
+
+      // PUSH NOTIFICATIONS
+      PRE_SHARED_KEY = "appbackend-PRE-SHARED-KEY"
+
+      // PAGOPA
+      ALLOW_PAGOPA_IP_SOURCE_RANGE : "appbackend-ALLOW-PAGOPA-IP-SOURCE-RANGE"
+
+      // TEST LOGIN
+      TEST_LOGIN_PASSWORD = "appbackend-TEST-LOGIN-PASSWORD"
+
+      // MYPORTAL
+      ALLOW_MYPORTAL_IP_SOURCE_RANGE : "appbackend-ALLOW-MYPORTAL-IP-SOURCE-RANGE"
+
+      // BPD
+      ALLOW_BPD_IP_SOURCE_RANGE : "appbackend-ALLOW-BPD-IP-SOURCE-RANGE"
+      JWT_SUPPORT_TOKEN_PRIVATE_RSA_KEY : "appbackend-JWT-SUPPORT-TOKEN-PRIVATE-RSA-KEY"
+    }
+  }
+
+  // TODO: Add ip restriction
+  allowed_ips = []
+
+  allowed_subnets = [
+    dependency.subnet_appgateway.outputs.id,
+    dependency.subnet_fn3services.outputs.id,
+    dependency.subnet_funcadmin_r3.outputs.id,
+  ]
+
+  subnet_id = dependency.subnet.outputs.id
+
+  application_logs = {
```

Comment:
are these values right for linux app services? (It looks like the are missing storage for logs)

Reply:
We will fix it with another PR.